main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -2977,7 +2978,7 @@ object Resolver {
       case Some(prefix) =>
         val ns = prefix ::: qname0.namespace.parts.tail
         val qname = Name.mkQName(ns, qname0.ident.name, SourceLocation.Unknown)
-        root.symbols.getOrElse(qname.namespace, Map.empty).get(qname.ident.name)
+        root.symbols.getOrElse(qname.namespace, Map.empty).get(qname.ident.name).map(_.distinct)
```

Comment:
This change may be significant?

Reply:
Why the extra map?

Reply:
Same as the other comment but for consistency. I did not observe any change in behavior on this line though so this could be removed maybe?